packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/TabZone/index.tsx

```diff
@@ -132,7 +135,7 @@ const TabZone: FC<TabZoneProps> = ({ tabId }) => {
       sectionId: tabId,
     });
 
-    setCustomType(newCustomType);
+    setCustomType(newCustomType, () => toast.success("Field added"));
```

Comment:
💡 #nit-pick

For consistency, I would use the same everywhere with or with the `{}` for the arrow function